test_requirements.txt

```diff
@@ -1,3 +1,4 @@
 -r requirements.txt
 coverage==3.7.1
 tox==1.8.1
+mock==1.0.1
```

Comment:
To be honest, I wouldn't add `mock` to these requirements. As I develop mostly on Python 3.4, `mock` is already in the stdlib. For running tests on older Python versions there's the `tox.ini` file which includes `mock` for some of the environments.